src/Codeception/Lib/Driver/Facebook.php

```diff
@@ -30,83 +42,107 @@ public function __construct($config, $logCallback = null)
      */
     public function createTestUser($name, array $permissions)
     {
+        $appToken = $this->appId . '|' . $this->appSecret;
         $response = $this->executeFacebookRequest(
-            $this->appSession,
             'POST',
-            '/' . FacebookSession::_getTargetAppId() . '/accounts/test-users',
+            $this->appId . '/accounts/test-users',
+            $appToken,
             [
                 'name' => $name,
                 'installed' => true,
                 'permissions' => $permissions
             ]
-        )->getRawResponse();
+        );
 
-        return json_decode($response, true);
+        return $response->getDecodedBody();
     }
 
     public function deleteTestUser($testUserID)
     {
+        $appToken = $this->appId . '|' . $this->appSecret;
         $this->executeFacebookRequest(
-            $this->appSession,
             'DELETE',
-            '/' . $testUserID
+            '/' . $testUserID,
+            $appToken
         );
     }
 
     public function getTestUserInfo($testUserAccessToken)
     {
         $response = $this->executeFacebookRequest(
-            new FacebookSession($testUserAccessToken),
             'GET',
-            '/me'
-        )->getRawResponse();
+            '/me',
+            $testUserAccessToken
+        );
 
-        return json_decode($response, true);
+        return $response->getDecodedBody();
```

Comment:
no, now only id and name are returned, and before it seems we had these properties:
id, first_name, gender, last_name, link, locale, name, timezone, updated_time and verified. It seems api has changed a bit here, although we are only using the 'name' property so that is why tests are passing.